libcontainer/cgroups/cgroups.go

```diff
@@ -32,6 +32,13 @@ type Manager interface {
 	// any all its sub-cgroups.
 	GetAllPids() ([]int, error)
 
+	// GetTids returns the TIDs of all processes inside the cgroup
+	GetTids() ([]int, error)
+
+	// GetAllTids returns the TIDs of all processes inside the cgroup
+	// any all its sub-cgroups.
+	GetAllTids() ([]int, error)
```

Comment:
Are these functions used by the runc CLI itself?
If not, probably https://github.com/containerd/cgroups is a better place to implement them.

Reply:
Thanks for your advice